beekeeper/src/main/resources/microservice.yaml

```diff
@@ -7,9 +7,9 @@ service_description:
 cse:
   service:
     registry:
-      address: http://sc.servicecomb.io:30100
```

Comment:
always think about automated tests instead of making changes to suit manually testing locally